src/Pages/Categories.js

```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>Categories</h1>
+);
+
```

Comment:
- [ ] `required`
- [ ] Great job so far 👏 
- [ ] however `Category` Should display a button "Check status" .
- [ ] Please fix it 🥂

Reply:
Thanks for the review, I'll do all the changes